torch/_inductor/pattern_matcher.py

```diff
@@ -743,6 +840,58 @@ def record_joint_graph(joint_graph, inputs, **kwargs):
     return gm
 
 
+def _args(n: torch.fx.Node):
+    args = list()
+    torch.fx.map_arg((n.args, n.kwargs), args.append)
+    return args
+
+
+def stable_topological_sort(graph: torch.fx.Graph):
```

Comment:
[nit] This is verifying rather than actually sorting? From the function name it looked like it would topologically sort the graph (something like reorder_for_locality)

Reply:
No, it is actually sorting.  The function name is correct.